ddtrace/contrib/grpc/patch.py

```diff
@@ -1,64 +1,85 @@
 import grpc
-from ddtrace.vendor import wrapt
 
-from ddtrace import Pin
-from ...utils.wrappers import unwrap
+from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
+from ddtrace import config, Pin
 
-from .client_interceptor import GrpcClientInterceptor
+from ...utils.wrappers import unwrap as _u
+
+from .client_interceptor import create_client_interceptor
+from .server_interceptor import create_server_interceptor
+
+
+config._add('grpc', dict(
```

Comment:
I can't tell if we need two configs... one for the client and one for the server service names to get the effect we are looking for.